tests/integration/test_storage.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `ebr_trackerbot` package."""
+
+import sys
+import os
+
+sys.path.append("ebr_trackerbot")
+sys.path.append("ebr_trackerbot/storage")
+
+from bot import config
+from state import STATE
+import pytest
+import sqlite
+import odbc
+import tempfile
+
+
+def delete_fixtures():
```

Comment:
Use the py.test fixtures instead: https://docs.pytest.org/en/latest/fixture.html?highlight=teardown#fixtures-as-function-arguments.

Will make the code cleaner

Reply:
The name is probably misleading. I don't need to pass some fixtures I need to prepare some files so I implemented setup and teardown functions.

Reply:
@bestdeveloperever I agree with @martinm82 here, because you can do a cleaner setup and teardown via fixtures. Take a look at https://docs.pytest.org/en/latest/fixture.html#fixture-finalization-executing-teardown-code
It should allow condensing these first three functions into a single function.

Reply:
But you can use py.test fixtures to achieve this kind of stuff. Imagine your test will create hundreds of file handles and throw an exception. The teardown won't be called to cleanup them

Reply:
but I see in documentation:
For tearDown(): “This method will only be called if the setUp() succeeds, regardless of the outcome of the test method.”
So if test throw exception then teardown function should be called

Reply:
This is integration test

Reply:
@bestdeveloperever you are referencing documentation from the unittest module but not py.test.
Right now what you do is a home-brew solution to do a setup and teardown. As @eugene-davis pointed out, there is a more elegant way of doing it.

Reply:
To correct what I was saying before. I was still having your initial version in mind when saying "homebrew". So the new version is better, but there is even a more elegant one.